src/main/java/games/strategy/engine/random/CryptoRandomSource.java

```diff
@@ -18,38 +18,38 @@
    * converts an int[] to a byte[].
    */
   public static byte[] intsToBytes(final int[] ints) {
-    final byte[] rVal = new byte[ints.length * 4];
+    final byte[] bytes = new byte[ints.length * 4];
     for (int i = 0; i < ints.length; i++) {
-      rVal[4 * i] = (byte) (0x000000FF & ints[i]);
-      rVal[(4 * i) + 1] = (byte) ((0x000000FF & (ints[i] >> 8)));
-      rVal[(4 * i) + 2] = (byte) ((0x000000FF & (ints[i] >> 16)));
-      rVal[(4 * i) + 3] = (byte) ((0x000000FF & (ints[i] >> 24)));
+      bytes[4 * i] = (byte) (0x000000FF & ints[i]);
+      bytes[(4 * i) + 1] = (byte) ((0x000000FF & (ints[i] >> 8)));
+      bytes[(4 * i) + 2] = (byte) ((0x000000FF & (ints[i] >> 16)));
+      bytes[(4 * i) + 3] = (byte) ((0x000000FF & (ints[i] >> 24)));
     }
-    return rVal;
+    return bytes;
```

Comment:
There are simpler ways to code this:
[See John Skeet's answer on stackoverflow](https://stackoverflow.com/a/1086092)

Reply:
Done in 52365a6.  Note that the legacy calculation is hard-coded to encode the `int`s in little endian order.